.gitignore

```diff
@@ -217,6 +217,29 @@ dmypy.json
 # Ignore all local history of files
 .history
 
+### PyCharm ###
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/dictionaries
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.xml
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/gradle.xml
+.idea/**/libraries
+.idea/misc.xml
+.idea/modules.xml
+.idea/vcs.xml
+.idea/jsLinters/jshint.xml
+.idea/inspectionProfiles/Project_Default.xml
+.idea/.gitignore
+.idea/inspectionProfiles/profiles_settings.xml
+.idea/django-fontawesome-5.iml
+*.iws /out/
```

Comment:
Please handle this type of personal stuff under your global gitignore. https://stackoverflow.com/a/7335487/1726238